rllib/algorithms/a2c/a2c.py

```diff
@@ -127,6 +127,8 @@ def validate_config(self, config: AlgorithmConfigDict) -> None:
                     "`microbatch_size` (try setting them to the same value)! "
                     "Otherwise, microbatches of desired size won't be achievable."
                 )
+        if config.get("num_gpus", 0) > 1:
+            raise ValueError("A2C with gpu learning enabled only supports 1 gpu.")
```

Comment:
nit nit nit picking man, should we say:

A2C does not support num_gpus > 1, because gradients are computed on the worker nodes.

so there is a little bit of "why" in the error message, in addition to "what".

Reply:
gotcha gotcha